lib/shoulda/matchers/active_record/define_enum_for_matcher.rb

```diff
@@ -84,6 +84,8 @@ def description
             desc << " with #{options[:expected_enum_values]}"
           end
 
+          desc << " and store the value in a column with an integer type"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.